vassal-app/src/main/java/VASSAL/build/AbstractConfigurable.java

```diff
@@ -259,4 +261,27 @@ public Configurer getConfigurer() {
   public List<String> getPropertyList() {
     return Collections.emptyList();
   }
+
+  /**
+   * Returns the name of the configurable type for display purposes. Reflection is
+   * used to call <code>getConfigureTypeName()</code>, which should be
+   * a static method if it exists in the given class. (This is necessary
+   * because static methods are not permitted in interfaces.)
+   *
+   * @return the configure name of the class
+   */
+  public String getTypeName() {
+    try {
+      return (String) this.getClass().getMethod("getConfigureTypeName").invoke(null);
+    }
+    catch (NoSuchMethodException e) {
+      // Ignore. This is normal, since some classes won't have this method.
+    }
+    catch (IllegalAccessException | ExceptionInInitializerError
+      | NullPointerException | InvocationTargetException | IllegalArgumentException e) {
+      ErrorDialog.bug(e);
+    }
+
+    return this.getClass().getName().substring(this.getClass().getName().lastIndexOf(".") + 1);
```

Comment:
I think you can use `this.getClass().getSimpleName()` here.

Reply:
Done